totalRP3/Modules/Register/Characters/ReportProfileButton.lua

```diff
@@ -31,8 +29,8 @@ TRP3_API.RegisterCallback(TRP3_Addon, TRP3_Addon.Events.WORKFLOW_ON_LOADED, func
 			end
 			Ellyb.Popups:OpenURL("https://battle.net/support/help/product/wow/197/1501/solution", reportText, nil, loc.COPY_SYSTEM_MESSAGE);
 		end,
-		tooltip =  REPORT_ICON:GenerateString(25) .. loc.REG_REPORT_PLAYER_PROFILE,
+		tooltip =  loc.REG_REPORT_PLAYER_PROFILE,
```

Comment:
```suggestion
		tooltip = loc.REG_REPORT_PLAYER_PROFILE,
```
The spacing is fucking raw.